dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java

```diff
@@ -60,11 +58,11 @@ public static FieldSql getInstance() {
 			+ "mod_date=? "
 			+ "where inode =?";
 	
-	public String updateFieldInode = "update inode set inode=?, idate=?, owner = ? where inode = ? and type='field'";
+	public final String updateFieldInode = "update inode set inode=?, idate=?, owner = ? where inode = ? and type='field'";
 	
-	public String insertFieldInode = "insert into inode (inode, idate, owner, type) values (?,?,?,'field')";
+	public final String insertFieldInode = "insert into inode (inode, idate, owner, type) values (?,?,?,'field')";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)